specification/nodes/_types/Stats.ts

```diff
@@ -196,6 +196,10 @@ export interface PressureMemory {
    * Number of indexing requests rejected in the replica stage.
    */
   replica_rejections?: long
+  primary_document_rejections?: long
+  limit?: ByteSize
+  limit_in_bytes?: long
```

Comment:
There's an `endObject()` before `limit_in_bytes`, suggesting it's not at the same level: https://github.com/elastic/elasticsearch/blob/c8e719544a9228c6b87a0d23100d478411101656/server/src/main/java/org/elasticsearch/index/stats/IndexingPressureStats.java#L342-L343